postreise/plot/plot_utilization_map.py

```diff
@@ -35,92 +20,109 @@ def map_risk_bind(
     vmin=None,
     vmax=None,
     is_website=False,
+    palette=None,
+    all_branch_scale_factor=5e-4,
+    all_branch_min_width=0.2,
+    select_branch_scale_factor=1e-3,
+    select_branch_min_width=2,
+    figsize=(1400, 800),
+    show_color_bar=True,
+    plot_states_kwargs=None,
 ):
     """Makes map showing risk or binding incidents on US states map.
 
     :param str risk_or_bind: specify plotting "risk" or "bind"
     :param pandas.DataFrame congestion_stats: data frame as returned by
         :func:`postreise.analyze.transmission.utilization.generate_cong_stats`.
     :param pandas.DataFrame branch: branch data frame.
-    :param dict us_states_dat: dictionary of state border lats/lons. If None, get
-        from :func:`postreise.plot.plot_states.get_state_borders`.
     :param int/float vmin: minimum value for color range. If None, use data minimum.
     :param int/float vmax: maximum value for color range. If None, use data maximum.
     :param bool is_website: changes text/legend formatting to look better on the website
-    :return:  -- map of lines with risk and bind incidents color coded
+    :param iterable palette: sequence of colors used for color range, passed as
+        `palette` kwarg to :func:`bokeh.transform.linear_cmap`.
+        If None, default to `postreise.plot.colors.traffic_palette`.
+    :param int/float all_branch_scale_factor: scale factor for unhighlighted branches.
+    :param int/float all_branch_min_width: minimum width for unhighlighted branches.
+    :param int/float select_branch_scale_factor: scale factor for highlighted branches.
+    :param int/float select_branch_min_width: minimum width for highlighted branches.
+    :param tuple(int, int) figsize: size of the bokeh figure (in pixels).
+    :param bool show_color_bar: whether to render the color bar on the figure.
+    :param dict plot_states_kwargs: keyword arguments to be passed to
+        :func:`postreise.plot.plot_states.plot_states`.
+    :return: (*bokeh.plotting.figure*) -- map of lines with risk and bind incidents
+        color coded.
     """
```

Comment:
Do we often change the `threshold` to calculate the `risk` (total power flowing on the line for hours used above  `threshold`) in `postreise.analyze.transmission.generate_cong_stats`? By default it is 0.75.

I believe the user experience would be simplified if we pass a `Scenario` object in place of `congestion_stats` and do:
```
grid = scenario.get_grid()
pf = scenario.get_pf()

congestion_stats = generate_cong_stats(pf, grid.branch)
```
Also, this would mirror what we do in other plotting routine (including the maps I have refactored in `ben/readme`), where we pass a scenario object.

Reply:
I'm agnostic to whether we pass a Scenario or the processed data. If someone wants to make multiple plots with the same set of processed results, then the calculation will be a bit repetitive, but it shouldn't matter too much.

Reply:
> I'm agnostic to whether we pass a Scenario or the processed data. If someone wants to make multiple plots with the same set of processed results, then the calculation will be a bit repetitive, but it shouldn't matter too much.

That's why some of the calculation heavy plots in PostREISE we have `areas` instead of `area` as parameter to produce  multiple plots in one shot using the same set of processed results. It would be nice that there is way to take advantage of cached data based on some indicators (so that it won't do so if it is now allowed).

Reply:
Done, the user can now pass either.